src/area-chart/__integ__/area-chart.test.ts

```diff
@@ -259,13 +297,32 @@ describe('Focus delegation', () => {
 
       await page.keys(['ArrowUp', 'ArrowRight', 'ArrowRight', 'Enter']);
 
-      await expect(page.getHighlightedSeriesLabel()).resolves.toBe('p60');
+      await expect(page.getHighlightedSeriesLabel()).resolves.toBe('p50');
       await expect(page.getPopoverTitle()).resolves.toBe('3s');
       await expect(page.isPopoverPinned()).resolves.toBe(true);
 
       await page.dismissPopover();
 
-      await expect(page.getHighlightedSeriesLabel()).resolves.toBe('p60');
+      await expect(page.getHighlightedSeriesLabel()).resolves.toBe('p50');
+      await expect(page.getPopoverTitle()).resolves.toBe('3s');
+      await expect(page.isPopoverPinned()).resolves.toBe(false);
+    })
+  );
+
+  test(
+    'when unpinning the popover the previously highlighted data point group is focused',
+    setupTest('#/light/area-chart/test', 'Linear latency chart', async page => {
+      await page.focusPlot();
+
+      await page.keys(['ArrowRight', 'ArrowRight', 'Enter']);
+
+      await expect(page.getHighlightedSeriesLabel()).resolves.toBe(null);
+      await expect(page.getPopoverTitle()).resolves.toBe('3s');
```

Comment:
After two `ArrowRight`, shouldn't this be `2s`?  Or am I missing something?

Reply:
After calling `await page.focusPlot()`, the focus is already on the `1s` because this function already types `ArrowRight` once. So after going right two more times we are at `3s`.